### PR TITLE
recreate tutorials in live playground, update links in Example.tsx

### PR DESCRIPTION
**Pull request by hichana**
Closes: #186

## Description
Issue #186 found that tutorial playgrounds contract deployment resulted in a blank screen and console error. After trying all of the tutorial playgrounds I found they all have the same issue. This PR has updated links in `Examples.tsx` with newly-created versions of each tutorial playground instance saved to the api via play.onflow.org. It resolves the issue of user's clicking on a tutorial and facing this issue when trying to deploy contracts or fork the project. 

Note, the underlying issue still needs to be investigated, as previously saved playgrounds that users may have might have the same issue. I will open a new issue to document that.

______

For contributor use:

- [x ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md))
- [x ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x ] Code follows the [standards mentioned here](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md).
- [n/a ] Updated relevant documentation 
- [x ] Re-reviewed `Files changed` in the Github PR explorer
- [x ] Added appropriate labels

### CODE REVIEW

File: src/components/Examples.tsx [hunk 1]
```diff
@@ -21,47 +21,47 @@ const examples = [
     subtitle:
       "Write your first contract on Flow. This is the perfect place to start to get the hang of the fundamentals of Cadence.",
     emoji: "🌎",
-    projectLink: "https://play.onflow.org/83315e10-8a9d-40d1-9f4a-877ca93eb93c",
+    projectLink: "https://play.onflow.org/d56be70d-33a2-4033-a6ab-38ed8c91aa76?type=account&id=0",
```
Comment (joshuahannan) [inline src/components/Examples.tsx:24]:
```suggestion
    projectLink: "https://play.onflow.org/f51905e8-6030-4641-9324-11a3f1a6091c",
```

File: src/components/Examples.tsx [hunk 2]
```diff
@@ -21,47 +21,47 @@ const examples = [
...
     docsLink: "https://docs.onflow.org/cadence/tutorial/02-hello-world/"
   },
   {
     title: "Mint Fungible Tokens",
     subtitle:
       "Create and sell digital assets of your own in this tutorial! This tutorial will teach you the basics of creating, storing, and moving digital assets and tokens.",
     emoji: "💸",
-    projectLink: "https://play.onflow.org/7c60f681-40c7-4a18-82de-b81b3b6c7915",
+    projectLink: "https://play.onflow.org/b402a10f-1a64-4f64-bd93-ce622f4562a7?type=account&id=0",
```
Comment (joshuahannan) [inline src/components/Examples.tsx:32]:
```suggestion
    projectLink: "https://play.onflow.org/50745fb6-77d5-4510-adfc-cf448fb043e1",
```

**Review by joshuahannan — APPROVED** [review]

**Comment by joshuahannan** [comment]
Thanks for updating these! I've been going through an updating the tutorials recently and didn't know about these links so the first two were out of date anyway so I've added the correct links here. I've got my PR open for the fungible token tutorial if you'd like to give it a review! https://github.com/onflow/cadence/pull/1238

**Comment by hichana** [comment]
Ok, links changed @joshuahannan !

**Review by hichana — COMMENTED** [review]
Thank you, all changed.

**Review by 10thfloor — APPROVED** [review]

**Pull request merged**